lib/osf-components/addon/components/infinite-list/template.hbs

```diff
@@ -4,7 +4,9 @@
         {{yield item to='item'}}
     {{/each}}
     {{#if @isLoading}}
-        <LoadingIndicator @dark={{true}} />
+        <li>
```

Comment:
Wrapped this in `<li>` as the parent `<ul>` should only have list items as direct children